src/package.json

```diff
@@ -1,22 +1,23 @@
 {
-  "name": "evergreen-node-system-function",
-  "version": "0.0.1",
-  "description": "A node function to integrate with riff and an evergreen function.",
+  "name": "node-function-buildpack",
```

Comment:
Did you mean to update this file? The name, license, and all that not pointing to this repo seems wrong.

Reply:
Yeah, that doesn't look right.  I'll clean this up.